src/components/Notification/Notification.tsx

```diff
@@ -4,32 +4,41 @@ import React, { useEffect, useRef } from "react";
 import type { HTMLProps, ReactNode } from "react";
 
 import Button, { ButtonAppearance } from "../Button";
-import type { Props as ButtonProps } from "../Button";
 
-export const notificationTypes = {
+import type { Values } from "types";
+
+export const NotificationAppearance = {
+  BORDERLESS: "borderless",
+  DEFAULT: "default",
+  MODAL: "modal",
+  RAISED: "raised",
+} as const;
```

Comment:
What does `export const ... as const` do? I didn't know this notation.

Reply:
The `as const` part tells the TS compiler that the object is completely static, otherwise the type inference for the values is `string` instead of very specific strings ("modal", "raised", etc.) . It's the way to get `enum`-like behaviour without having to use `enum`s.

Reply:
Thanks @Caleb-Ellis, it's a bit weird to read/write in code, but I like the concept of having strict list of strings, but also being able to pass a string as a value, not just enum.

I'll let you discuss with @huwshimi if that's the direction we want to go with values like this.